server-spi-private/src/main/java/org/keycloak/authorization/policy/evaluation/DefaultEvaluation.java

```diff
@@ -102,4 +124,128 @@ void denyIfNoEffect() {
             deny();
         }
     }
+
+    private Realm createRealm() {
+        return new Realm() {
+
+            @Override
+            public boolean isUserInGroup(String id, String groupId, boolean checkParent) {
+                KeycloakSession session = authorizationProvider.getKeycloakSession();
+                UserModel user = getUser(id, session);
+
+                if (Objects.isNull(user)) {
```

Comment:
While this can be just a technicality, Objects.isNull() exists so it can be used as a Predicate. I don't see any major advantage of using it here instead of the (user == null) kind of check. Just pointing this out although I'm fine with leaving it as is.

Reply:
Compiler is probably going to replace this with a ``user == null``. If you are OK with it I would just leave as it is.

Reply:
Its fine